src/_posts/awoa-06.md

```diff
@@ -0,0 +1,25 @@
+---
+title: A Week of A-Frame 6
+date: 2016-02-26
+layout: blog
+
+awoa:
+  projects:
+    - title: Living Room
+      author: github|dkraeker
+      description: WebVR and chill.
+      image: living-room.png
+      url: http://drawvr.com/living-room/
+
+    - title: JM Visual Creativity
+      author: twitter|chetemele
+      description: Web designer and motion graphics artist Jose Manuel Serna's portfolio.
+      image: jm-visual-creativity.png
+      url: http://vr.jmvisualcreativity.es/
+
+    - title: SpineFrame
+      author: github|joshgalvin
+      description: Spine skeletal animation with a spine component that uses spine-threejs.
+      image: spineframe.png
+      url: https://joshgalvin.github.io/SpineFrame/examples/basic
```

Comment:
nvm, that was me being clever by stripping the `.html` from the URL because GH Pages lets you